wrangler-core/src/main/java/co/cask/wrangler/steps/Expression.java

```diff
@@ -95,6 +97,8 @@ public Expression(int lineno, String detail, String column, String expression) {
     functions.put(null, Convertors.class);
     functions.put("math", Math.class);
     functions.put("string", StringUtils.class);
+    functions.put("bytes", Bytes.class);
+    functions.put("arrays", Arrays.class);
```

Comment:
The idea is to test for directives to access the functionality. Don't need to test each functions within arrays.